src/main/scala/cromwell/util/ConfigUtil.scala

```diff
@@ -26,14 +26,20 @@ object ConfigUtil {
 
     def keys = config.entrySet().toSet map { v: java.util.Map.Entry[String, ConfigValue] => v.getKey }
 
-    def getStringOption(key: String): Option[String] = {
```

Comment:
we need to shelve lenthalliation for the time being unless it's close to being a no op.